app/controllers/email_verification_controller.rb

```diff
@@ -0,0 +1,17 @@
+class EmailVerificationController < ApplicationController
+  def edit
+  end
+
+  def update
+    user = User.find_by(verification_token: params[:verification_token])
+
+    if user
+      user.update_attributes(verified_email: true)
+      session[:user_id] = user.id
```

Comment:
Looks good.  I think the session[:user_id] assignment will be necessary in case the user logs out before verifying their account.